packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -9,6 +9,9 @@
     "participantStateRinging": "Calling...",
     "participantStateHold": "On hold"
   },
+  "ParticipantList": {
+    "overflowParticipantCount": "+{overflowCount} more"
```

Comment:
I am curious if we can review the use of "more" im unsure how its going to scale when we think of all of the other localizations